railties/test/generators/app_generator_test.rb

```diff
@@ -534,6 +534,10 @@ def test_generator_if_skip_test_is_given
     assert_no_gem "selenium-webdriver"
 
     assert_no_directory("test")
+
+    assert_file ".github/workflows/ci.yml" do |file|
+      assert_no_match /test\:.\s*runs\-on/m, file
+    end
```

Comment:
@skipkayhil how does this look? I reverted my changed in `railties/lib/rails/generators/rails/app/templates/github/ci.yml.tt` and got a failure.

Reply:
Also fixed a linting violation.